src/test/java/com/artipie/front/settings/RepoDataTest.java

```diff
@@ -70,9 +72,15 @@ void removesData() {
 
     @Test
     void movesData() {
+        this.stngs.save(
+            new Key.From(String.format("%s.yml", RepoDataTest.REPO)),
+            this.repoSettings().getBytes(StandardCharsets.UTF_8)
+        );
+        this.data.save(new Key.From(RepoDataTest.REPO, "first.txt"), new byte[]{});
+        this.data.save(new Key.From(RepoDataTest.REPO, "second.txt"), new byte[]{});
         final String nrepo = "new-repo";
-        new RepoData(new RepoSettings("flat", this.stngs)).move(RepoDataTest.REPO, "any", nrepo)
-            .toCompletableFuture().join();
+        new RepoData(new RepoSettings("flat", this.stngs))
+            .move(RepoDataTest.REPO, "any", nrepo).toCompletableFuture().join();
         MatcherAssert.assertThat(
             "Repository data are moved",
```

Comment:
Should we check that the old repository data doesn't exist?

Reply:
same here